src/types/onyx/Paypal.ts

```diff
@@ -0,0 +1,27 @@
+import CONST from '../../CONST';
+import * as OnyxCommon from './OnyxCommon';
+
+type PaypalAccountData = {
+    username?: string;
```

Comment:
```suggestion
    username: string;
```

I think this should be required since it is the only param in the object.

Reply:
Have you checked it with the backend that it is indeed required? @madmax330

Reply:
Yes

Reply:
Thanks!